documentation/STRUCTURE.md

```diff
@@ -0,0 +1,161 @@
+
+All the case studies should be downloadable as Jupyter notebooks and/or Python source code, and be launchable in [Binder](https://mybinder.org/) and perhaps other platforms.
+
+Note: [scikit-learn](https://scikit-learn.org/dev/auto_examples/index.html) refers to these as "Examples".
+
+Of our current notebooks the following would be most closely aligned with this section:
+
+- [Binary Classification on COMPAS dataset](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Binary%20Classification%20on%20COMPAS%20dataset.ipynb) - although we should perhaps consider removing it since it may not do justice to this complex setup
+- [Binary Classification with the UCI Credit-card Default Dataset](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Binary%20Classification%20with%20the%20UCI%20Credit-card%20Default%20Dataset.ipynb)
+- [Mitigating Disparities in Ranking from Binary Data](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Mitigating%20Disparities%20in%20Ranking%20from%20Binary%20Data.ipynb)
+
+### Community
+
+This page should provide basic information about how to reach out in order to ask questions, provide feedback, report bugs or request features. Similarly, it should be clear how people can reach out if they want to contribute to Fairlearn, and where they can find small items to get started. [This](https://scikit-learn.org/dev/developers/contributing.html) is an example of how scikit-learn handles it through a contribution guide that is somewhat similar to ours that we currently have in the repo.
+
+We could also write up the history of the project, its roadmap, and how we envision governance to work. The code of conduct should certainly be mentioned as well.
+
+Some projects have a page showing the maintainers as well:
+
+- [scikit-learn](https://scikit-learn.org/stable/about.html#people)
+- [pandas](https://pandas.pydata.org/about/team.html)
+
+## Required steps
+
+To properly format math, text, and links across the pages we need to use ReST. Some of our existing documentation may have to be converted from Markdown to ReST, e.g. the terminology page.
+
+We need to find a way to present the dashboard in a website where it can't be interactive. Perhaps with screenshots for the user guides, but the case studies are downloadable as Jupyter notebooks. [Could we perhaps pre-calculate all metrics and show the interactive dashboard in the case studies?]
```

Comment:
you could also proactively mention that the user can go to the notebook platform (e.g. binder) with a click and see them live.

Reply:
Definitely.

The interactivity option also has potential, although the precomputed statistics would have to be small, and we'd need a way of automatically updating them. Plus, the dashboard would need tweaking and we'd have to get the serialisation format properly nailed down (it's very ad hoc right now). Those are all solvable problems; there could also be security issues, since it would be a large stack of JS being injected into user's browsers.